bitrise.yml

```diff
@@ -483,6 +497,12 @@ workflows:
     - slack@3.1:
         inputs:
         - webhook_url: $WEBHOOK_SLACK_TOKEN
+    - slack@3.1:
+        run_if: '{{getenv "NEW_AS_VERSION" | eq "New_AS_Version"}}'
+        inputs:
+        - channel: "#firefox-ios"
+        - text: Build status using latest A-S
+        - webhook_url: $WEBHOOK_SLACK_TOKEN_2
```

Comment:
Just double checking on ```$WEBHOOK_SLACK_TOKEN_2``` 2?

Reply:
yeah, that's the token for firefox-ios channel, I should change the name on bitrise or document that... the regular WEBHOOK_SLACK_TOKEN is the one for firefox-ios-alerts